tt_metal/include/compute_kernel_api/bcast.h

```diff
@@ -150,10 +150,10 @@ ALWI void any_tiles_bcast(uint32_t icb0, uint32_t icb1, uint32_t itile0, uint32_
  * For *dim==BroadcastType::COL*, the input in *B* is expected to be a single tile with a
  * filled 0-column and zeros elsewhere.  The result is *C[h, w] = A[h,w] + B[w]*
  *
- * For *dim==Dim::C*, the input in *B* is expected to be a single tile with a
+ * For *dim==VectorMode::C*, the input in *B* is expected to be a single tile with a
```

Comment:
The changes to this file are incorrect. The enum is BroadcastType with values ROW, COL, or SCALAR, not VectorMode.